forge-gui/res/cardsfolder/upcoming/badlands_revival.txt

```diff
@@ -0,0 +1,7 @@
+Name:Badlands Revival
+ManaCost:3 B G
+Types:Sorcery
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Choose up to one target creature card in your graveyard | ValidTgts$ Creature.YouOwn | SubAbility$ DBReturn | SpellDescription$ Return up to one target creature card from your graveyard to the battlefield. 
+SVar:DBReturn:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Choose up to one target permanent card in your graveyard | ValidTgts$ Card.Permanent+YouOwn | SpellDescription$ Return up to one target permanent card from your graveyard to your hand.
```

Comment:
It doesn't let me choose targets for this. Thoughts ?

Reply:
On my side, I just click on the graveyard (again) to bring it up so I can select the permanent to return to hand. It's maybe by design that I can select the creature I chose for the first part of the spell ability again for the second one. 
![image](https://github.com/Card-Forge/forge/assets/45150760/d39eec2e-9866-428a-88ae-4c5981978b71)

Reply:
I had a look and [Crop Sigil](https://scryfall.com/card/emn/153/crop-sigil)'s activated ability works the same way: you need to click your graveyard to be able to select the land to return to hand after you've picked the creature. I remember having a similar issue with [Sludge Titan](https://scryfall.com/card/clu/43/sludge-titan) but unfortunately there the cards aren't targeted so the `EACH` tag proposed by @tool4ever to resolve it might not be applicable here especially considering the cards go to different zones.

Reply:
Thanks for testing and you're right that it works but you will have to open your graveyard first